sourced/ml/tests/test_bblfsh_utils.py

```diff
@@ -0,0 +1,85 @@
+import docker.client
+import errno
+import random
+import socket
+import time
+import unittest
+
+from sourced.ml.utils.bblfsh import check_version, BBLFSH_VERSION_LOW, BBLFSH_VERSION_HIGH
+
+
+def check_bblfsh_version_support(client: docker.client.DockerClient, version: str) -> bool:
+    """
+    :param client: docker client instance
+    :param version: version of bblfshd to check
+    :return: True if version is supported, False otherwise
+    """
+    s = socket.socket()
+    for i in range(3):
+        try:
+            port = random.randint(10000, 50000)
+            s.connect(("localhost", port))
+        except socket.error as e:
+            if e.errno == errno.ECONNREFUSED:
+                break
+    s.close()
+
+    container = client.containers.run(
+        image="bblfsh/bblfshd:%s" % (version),
+        privileged=True,
+        detach=True,
+        ports={"9432": port},
+    )
+
+    assert container is not None, "failed to create bblfsh container"
+
+    for i in range(3):
+        try:
+            res = check_version(port=port)
+            break
+        except Exception as e:
+            time.sleep(5)
+            pass
+
+    container.stop()
+    container.remove()
+    return res
+
+
+class BblfshUtilsTests(unittest.TestCase):
```

Comment:
Please put the tests into a separate file

Reply:
can I just take the `check_bblfsh_version_support` and make it a method in `BblfshUtilsTests`? I don't see the reason to split it into separate files, because that method is the test itself.

Reply:
Sorry, I thought this was a normal source code file. Ignore this pls.